test/server/cards/01_SOR/units/CargoJuggernaut.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Cargo Juggernaut', function () {
+    integration(function () {
+        describe('Cargo Juggernaut\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['cargo-juggernaut'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors']
+                    }
+                });
+            });
+
+            it('should heal base', function () {
+                this.p1Base.damage = 10;
+                this.player1.clickCard(this.cargoJuggernaut);
+                this.player1.clickPrompt('If you control another [Vigilance] unit, heal 4 damage from your base');
+                expect(this.player2).toBeActivePlayer();
+                expect(this.p1Base.damage).toBe(6);
+            });
+
+            it('should not heal base', function () {
+                this.p1Base.damage = 10;
+                this.player1.passAction();
+                // kill our vigilance unit
+                this.player2.clickCard(this.ruggedSurvivors);
```

Comment:
Let's just create a second test setup with `describe` which does not have this unit present to make the test itself simpler and we don't have to do this. Since each test case has to build the whole game from start, it is does not make a difference in terms of test time if we add more `describe` blocks